apps/ledger-live-desktop/src/renderer/screens/settings/sections/Developer/ExchangeTestPartnerMode.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from "react";
+import { setEnv, getEnv } from "@ledgerhq/live-env";
+import Switch from "~/renderer/components/Switch";
+import Box from "~/renderer/components/Box";
+import { useTranslation } from "react-i18next";
+import { SettingsSectionRow } from "../../SettingsSection";
+
+const ExchangeTestPartnerMode = () => {
+  const { t } = useTranslation();
+  const [enableExchangeTestPartnerMode, setEnableExchangeDevMode] = useState<boolean>(false);
+
+  useEffect(() => {
+    setEnableExchangeDevMode(getEnv("MOCK_EXCHANGE_TEST_PARTNER"));
+  }, []);
```

Comment:
```suggestion
  const [enableExchangeTestPartnerMode, setEnableExchangeDevMode] = useState<boolean>(getEnv("MOCK_EXCHANGE_TEST_PARTNER"));
```
I think that you do not need this useEffect, am I wrong?